### PR TITLE
New footer

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Add required Vanilla V1 base, pattern, settings and utilities SASS files 
Update footer.html as per new designs and using V1 classes where possible

Does not include global nav atm although the SASS and js from the project have been included

The following files have been copied and pasted from Vanilla V1

_patterns_footer.scss
_patterns_links.scss
_patterns_lists.scss
_settings_colors.scss
_settings_grid.scss
_utilities_clearfix.scss
_utilities_margin_collapse.scss

## QA

Run ./run to get the site running

Check in s/m & l

## Issue / Card

Card: https://trello.com/c/ZhKJTn8J
Designs: https://www.dropbox.com/sh/65nknx3h5fqpvc3/AADRvxrQ-77m2utKAQJHMtP9a?dl=0

### CODE REVIEW

File: static/css/modules/_ubuntu_patterns_footer.scss [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    }
+    
+    &__title {
+      @include footer-title;
+      border-top: 1px solid $color-mid-light;
+      background-image: url("#{$asset-server-url}7bd1bd7b-arrow_down_9fa097.svg");
+      background-position: 96% 50%;
+      background-repeat: no-repeat;
+      background-size: 13px 13px;
+      font-size: .857142857em;
```
Comment (anthonydillon) [inline static/css/modules/_ubuntu_patterns_footer.scss]: rem please.

File: static/css/modules/_ubuntu_patterns_footer.scss [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+      padding: 10px 0;
+
+      @media only screen and (min-width: $navigation-threshold) {
+        background: none;
+        border: 0;
+        color: $color-dark;
+        font-size: 1em;
```
Comment (anthonydillon) [inline static/css/modules/_ubuntu_patterns_footer.scss]: rem please

File: static/css/modules/_ubuntu_patterns_footer.scss [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+    margin: 0;
+    
+    li {
+      margin-bottom: 0;
+      
+      a {
+        border-top: 1px solid $color-mid-light;
+        color: $color-dark;
+        display: block;
+        font-size: .8125em;
```
Comment (anthonydillon) [inline static/css/modules/_ubuntu_patterns_footer.scss]: rem please.

File: static/css/modules/_ubuntu_patterns_footer.scss [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+        padding: 10px;
+        
+        @media only screen and (min-width: $navigation-threshold) {
+          border: 0;
+          font-size: 1em;
```
Comment (anthonydillon) [inline static/css/modules/_ubuntu_patterns_footer.scss]: rem please

File: static/css/modules/_ubuntu_patterns_footer.scss [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+          padding: 0 0 5px 0;
+        }
+        
+        @media only screen and (min-width: $breakpoint-large) {
+          font-size: .8125em;
```
Comment (anthonydillon) [inline static/css/modules/_ubuntu_patterns_footer.scss]: rem please.

File: static/css/modules/_ubuntu_patterns_footer.scss [hunk 6]
```diff
@@ -0,0 +1,217 @@
...
+    @media only screen and (min-width: $navigation-threshold) {
+      padding: 0; 
+    }
+    
+    a:hover {
+      color: currentColor;
+    }
+    
+    &__content {
+      font-size: 1em;
```
Comment (anthonydillon) [inline static/css/modules/_ubuntu_patterns_footer.scss]: rem please

File: static/css/modules/_ubuntu_patterns_footer.scss [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+    }
+    
+    .button--primary {
+      color: $color-dark;
+      background-color: $white;
```
Comment (anthonydillon) [inline static/css/modules/_ubuntu_patterns_footer.scss]: `$color-x-light`

File: static/css/modules/_ubuntu_patterns_footer.scss [hunk 8]
```diff
@@ -0,0 +1,217 @@
...
+      border: 1px solid $color-mid-light;
+      padding: 6px 18px;
+  
+      &:hover {
+        background-color: #f7f7f7;
```
Comment (anthonydillon) [inline static/css/modules/_ubuntu_patterns_footer.scss]: `$color-light`

File: static/css/modules/_ubuntu_patterns_footer.scss [hunk 9]
```diff
@@ -0,0 +1,217 @@
...
+        
+        @media only screen and (min-width: $navigation-threshold) {
+          display: inline-block;  
+        }
+      }
+    }
+  }
+
+  // social icons in footer
+  .p-inline-list-icons {
```
Comment (anthonydillon) [inline static/css/modules/_ubuntu_patterns_footer.scss]: Could you use inline-list? https://docs.vanillaframework.io/en/patterns/lists#inline-list

File: templates/templates/footer.html [hunk 10]
```diff
@@ -1,82 +1,107 @@
...
-      <li><a href="https://github.com/ubuntudesign/www.ubuntu.com/issues/new" id="report-a-bug">Report a bug on this site</a></li>
-    </ul>
+  
+  <div class="strip-inner-wrapper strip-inner-wrapper--nav">
+    <div class="p-footer--secondary">
+      <div class="seven-col">
+        <nav>
+          <ul class="p-inline-list--small">
+            <li class="for-large p-inline-list__item"><a class="button--primary" href="/about/contact-us">Contact us</a></li>
+            <li class="not-for-large p-inline-list__item"><a class="p-link--soft" href="/about/contact-us">Contact us</a></li>          <li class="p-inline-list__item"><a class="p-link--soft" href="/about">About us</a></li>
```
Comment (anthonydillon) [inline templates/templates/footer.html]: Return line at the end of `</li>`

Comment (anthonydillon) [inline templates/templates/footer.html]: Two contact us

File: templates/templates/footer.html [hunk 11]
```diff
@@ -1,82 +1,107 @@
...
+            <li class="p-inline-list__item"><a class="p-link--soft" href="/http://www.canonical.com/careers">Careers</a></li>
+            <li class="p-inline-list__item"><a class="p-link--soft" href="https://insights.ubuntu.com/press-centre/">Press centre</a></li>
+          </ul>
+        </nav>
+        
+        <p class="p-footer--secondary__content">&copy; {% now "Y" %} Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.</p>
+        <nav>
+          <ul class="p-inline-list--middot-small u-no-margin--bottom">
+            <li class="p-inline-list__item"><a class="p-link--soft" accesskey="8" href="/legal">Legal information</a></li>
+            <li class="p-inline-list__item"><a class="p-link--soft" href="https://github.com/ubuntudesign/www.ubuntu.com/issues/new" id="report-a-bug">Report a bug on this site</a></li>
```
Comment (anthonydillon) [inline templates/templates/footer.html]: `canonical-websites/www.ubuntu.com`

**Comment by anthonydillon** [comment]
The middot list is a little low:

**Comment by anthonydillon** [comment]
Looks like it's missing the drop shadows instead of borders.

File: static/css/modules/_ubuntu_patterns_footer.scss (hunk 1 above)
Comment (grahambancroft) [inline static/css/modules/_ubuntu_patterns_footer.scss]: Done

File: static/css/modules/_ubuntu_patterns_footer.scss (hunk 2 above)
Comment (grahambancroft) [inline static/css/modules/_ubuntu_patterns_footer.scss]: Done

File: static/css/modules/_ubuntu_patterns_footer.scss (hunk 3 above)
Comment (grahambancroft) [inline static/css/modules/_ubuntu_patterns_footer.scss]: Done

File: static/css/modules/_ubuntu_patterns_footer.scss (hunk 4 above)
Comment (grahambancroft) [inline static/css/modules/_ubuntu_patterns_footer.scss]: Done

File: static/css/modules/_ubuntu_patterns_footer.scss (hunk 5 above)
Comment (grahambancroft) [inline static/css/modules/_ubuntu_patterns_footer.scss]: Done

**Comment by anthonydillon** [comment]
To much space around the top and bottom of the list on medium screen:

File: static/css/modules/_ubuntu_patterns_footer.scss (hunk 6 above)
Comment (grahambancroft) [inline static/css/modules/_ubuntu_patterns_footer.scss]: Done

File: static/css/modules/_ubuntu_patterns_footer.scss (hunk 7 above)
Comment (grahambancroft) [inline static/css/modules/_ubuntu_patterns_footer.scss]: Done

File: static/css/modules/_ubuntu_patterns_footer.scss (hunk 8 above)
Comment (grahambancroft) [inline static/css/modules/_ubuntu_patterns_footer.scss]: Done

File: templates/templates/footer.html (hunk 11 above)
Comment (grahambancroft) [inline templates/templates/footer.html]: Done

File: templates/templates/footer.html (hunk 10 above)
Comment (grahambancroft) [inline templates/templates/footer.html]: Return done. One of the contacts is a button for medium + so is hidden on small

**Comment by grahambancroft** [comment]
@anthonydillon All done and ready for re-review. Re the 'Looks like it's missing the drop shadows instead of borders.’ it looks like a border on the design, I’ll check with Joana

**Pull request merged**